package.json

```diff
@@ -37,14 +37,16 @@
         "Visualization"
     ],
     "activationEvents": [
+        "onLanguage:kusto",
         "onNotebook:kusto-notebook",
         "onNotebook:jupyter-notebook",
         "onLanguage:plaintext",
         "onView:kustoExplorer",
-        "onCommand:kusto.addCluster",
-        "onCommand:kusto.removeCluster",
+        "onCommand:kusto.executeSelectedQuery",
+        "onCommand:kusto.addConnection",
+        "onCommand:kusto.removeConnection",
         "onCommand:kusto.refreshNode",
-        "onCommand:kusto.changeNotebookDatabase"
+        "onCommand:kusto.changeDocumentConnection"
```

Comment:
Refactored to add support for more types of connecetions, hence renamed to `Connection`.